### DISCUSSION TITLE
Docker version

### DISCUSSION THREAD

**Discussion by gueulderack** [timoschwarzer/gitlab-monitor#202] [General]
Hi,
is it normal that no new version appear in docker hub since 1 year ago? I'm deploying you beautiful tool in version unprivileged by using docker installation, but I'm feared to see that nothing was updated since 1 year. Is it means that you have stopped maintaining docker images?

In all the case, thanks a lot.

**Comment by timoschwarzer** [comment]
You can find the latest version in the GitHub package registry. https://github.com/timoschwarzer/gitlab-monitor/pkgs/container/gitlab-monitor

The readme needs to be updated...

**Reply by gueulderack** [reply]
Ok, thank you. And what about the unpriviledged version?
